perfect_numbers.py

```diff
@@ -9,13 +9,12 @@ def all_perfect_numbers():
     while True:
         if sum(number for number in range(1, up_to) if not up_to % number) == up_to:
```

Comment:
What is up_to? up_to what? give your variables better names (apply this to number as well).